microsoft-edge/webview2/how-to/remote-debugging-xbox.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Remote debugging Xbox WebView2 WinUI 2 (UWP) apps
+description: How to remotely debug Xbox WebView2 WinUI 2 (UWP) apps by using Remote Tools for Microsoft Edge. 
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 09/27/2023
+---
+# Remote debugging Xbox WebView2 WinUI 2 (UWP) apps
+
+To use [Microsoft Edge DevTools](/microsoft-edge/devtools-guide-chromium/landing/) to debug a WebView2 WinUI 2 (UWP) app, use remote debugging.  Remote debugging is necessary for WebView2 WinUI 2 (UWP) apps because currently, the built-in DevTools can't be launched inside a store-signed WebView2 WinUI 2 (UWP) app.
+
+
+<!-- ========================================================================== -->
+## Attach DevTools remotely to an Xbox WebView2 WinUI 2 (UWP) app
+
+Attach Microsoft Edge DevTools remotely to an Xbox WebView2 WinUI 2 (UWP) app as follows:
+
+1.  In Microsoft Edge, go to `edge://inspect`.  The **Inspect with Edge Developer Tools** page opens:
+
+    ![DevTools Inspect utility page](./remote-debugging-xbox-images/inspect-devtools-page-supported.png)
+
+1.  Enable Developer Mode on Xbox.  See [Xbox One Developer Mode activation](/windows/uwp/xbox-apps/devkit-activation).
+
+1.  In Microsoft Edge, go to `https://<System IP>:11443`, where `<System IP>` is replaced by your Xbox System OS IP.  The **Confirm security settings** page is displayed, in the **Windows Device Portal** tab:
+
+    ![The 'Confirm security settings' page in the 'Windows Device Portal' tab](./remote-debugging-xbox-images/open-device-portal.png)
+
+    An SSL cert error appears:
+
+    ![SSL Error](./remote-debugging-xbox-images/xbox_ssl_error.png)
+
+1.  Click the **Advanced** button. <!--todo: state the system result-->
+
+1.  Click **Continue**<!--todo: button?--> to `<System IP>`,<!--todo: clarify "Click Continue to System IP"--> where `<System IP>` is replaced by your Xbox System OS IP.
```

Comment:
@maurawinstanley - clarify the formatting/wording around this step.  Check nearby screenshots in rendered section/article:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/remote-debugging-xbox?branch=pr-en-us-2846#attach-devtools-remotely-to-an-xbox-webview2-winui-2-uwp-app